### PR TITLE
Drop experimental tag from stabilized feature documentation.

### PR DESCRIPTION
**Pull request by epgts**
Closes issue #220.

### CODE REVIEW

File: docs/timeseries.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Timevector [<sup><mark>experimental</mark></sup>](/docs/README.md#tag-notes)
```
Comment (rtwalker) [inline docs/timeseries.md:1]: Is this stabilized? https://github.com/timescale/timescaledb-toolkit/issues/483

Comment (WireBaron) [reply docs/timeseries.md:1]: Timevectors are stabilized, timevector pipelines are not

File: docs/timeseries_pipeline_elements.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Timevector Pipelines [<sup><mark>experimental</mark></sup>](/docs/README.md#tag-notes)
```
Comment (WireBaron) [inline docs/timeseries_pipeline_elements.md:1]: This is a bit different than the pipelined accessors that we stabilized in 1.10.1.  I think this should still be marked experimental.  Hmm, taking a look this doc also looks terribly out of date as it only has three pipeline elements (I think we're closer to dozen at this point).

**Review by WireBaron — APPROVED** [review]

**Pull request merged**